package.json

```diff
@@ -34,9 +34,12 @@
     "zone.js": "0.6.21"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "0.0.39",
+    "@ionic/app-scripts": "0.0.41",
     "typescript": "^2.0.3"
   },
+  "config": {
+    "ionic_bundler": "webpack"
```

Comment:
Actually, this should be set to rollup to get the slightly smaller and faster bundle that it provides.